src/get-api-credentials.js

```diff
@@ -0,0 +1,19 @@
+import { ProjectCredentialsConfig } from 'sphere-node-utils'
+import Promise from 'bluebird'
+
+const getApiCredentials = (projectKey, accessToken) => {
+  if (!projectKey)
+    return Promise.reject(new Error('Project Key is needed'))
+
+  if (accessToken)
+    return Promise.resolve({ project_key: projectKey })
+
+  return ProjectCredentialsConfig.create()
```

Comment:
So I looked at the `sphere-node-utils` and I saw that, it exported a class, I guess the reason why the name is capitalised, but it's used directly without instantiating a new object of the class.

I guess we should update it. Easy change I guess.

Also, I know that it was used like this in other previous projects...

Also, looking at node-utils, it's the default module so doing 
```js
import ProjectCredentialsConfig from 'sphere-node-utils'
```
should work fine.